sql/mob_groups.sql

```diff
@@ -9722,7 +9722,7 @@ INSERT INTO `mob_groups` VALUES (31,4526,146,'Zuu_Xowu_the_Darksmoke',0,128,0,0,
 INSERT INTO `mob_groups` VALUES (32,1601,146,'Gii_Jaha_the_Raucous',0,128,0,0,0,60,60,0);
 INSERT INTO `mob_groups` VALUES (33,15,146,'Aa_Nawu_the_Thunderblade',0,128,0,0,0,60,60,0);
 INSERT INTO `mob_groups` VALUES (34,4474,146,'Yoo_Mihi_the_Haze',0,128,0,0,0,60,60,0);
-INSERT INTO `mob_groups` VALUES (35,4385,146,'Wyrm',0,128,0,45000,15000,85,85,0);
+INSERT INTO `mob_groups` VALUES (35,4385,146,'Wyrm',0,128,0,50000,15000,80,80,0);
```

Comment:
I haven't really examined the Early Bird capture, but be forewarned that if this is based on what HP Track spit out, if the mob has regen the numbers might not be accurate. The accuracy could also vary depending on the estimation method it needed to resort to.

Min and Max will just be the minimum and maximum damage it took to kill the mob, so if there was regen involved, the values will be higher than what the mob's HP truly is, as HP Track has no way of knowing DoT or Regen amounts.

Method I stands for Method Interval, in which it attempts to guess an HP based off of percentage differences it saw the mob's HP % change by, and the damage it took to reach those differences. Because it guesses HP in slices over the entire fight and then averages those guesses together, I consider its estimation more trustworthy than...

Method L stands for Method Last. It compares how much HP% the mob had when it was last seen alive, compared to how much damage the mob has taken over the entire battle before the killing blow. If HP Track last saw the mob alive at 3% after seeing exactly 97000 damage dealt, it'll spit out an estimate of 100,000 HP. However, since retail only sends out HPP updates for whole integers, the mob's true HP range could be 99,000~101,000. In this case we could guess SE selected 100,000, but if the exact damage dealt values, and the original mob's max HP value isn't as "clean", this can be harder to guess.

I consider Method Ls slightly less trustworthy because it only examines the battle at one point in time, _and_ while it may sound like a conspiracy theory, I'm not convinced 1% HP is _actually_ 1% HP. I _swear_ the last 1% takes longer to empty on retail than it "should". So if the last time it saw the mob alive was at this "1%" value, then it can come up with higher numbers if the mob really had more of its True HP remaining during "the last 1%" phase.

Obviously retail is going to round up if the mob is _sub_ 1%, which will lead to further inaccuracies. In HP Track's case, the damage it thinks it took to bring a mob down to "1%" (in this case, the total damage it's seen and is assuming is 99% of the mob's HP) could actually be anywhere from 98.1% (mob had 1.9% remaining) to 99.9% (mob had 0.1% remaining).

Reply:
Wow, after reading this I'm now more confused than before even though I appreciate the insight. I do get that HP regeneration might mess up readings though. You're very deep into the whole capturing so, I'd appreciate your instructions on this. It sounds like it would be sensible to revert this back to the 45,000 it had before.

Reply:
Looked at the capture:
`46728~51180 HP, Est HP: 49644 (Mthd: L)`
Method L is a little odd in that it compares the total damage it saw (max) to how much HP% it watched get drained. This is... actually not accurate, so I should fix that. It should be comparing the _minimum_ against what it watched get drained.

But we can use the min HP value to look at the problem from the other direction:
The final hit was a Fell Cleave taking off the final 3% of its HP.

It saw 46727 damage done up to that point. The "minimum HP" is "total damage and still living + 1 HP".
```
(Max HP/100) * 97 = 46727
Max HP / 100 = 481.72
Max HP = 48,172
```

Basically, even pretending the Regen never existed, if 46,727 damage brought it down to 3% HP left, then it can't have more than 48,172 HP.

Even if we assumed that the HP% that retail sends out is _floored_:
(Max HP/100) * 96.1 = 46727 (Note: 100 - 96.1 = 3.9)
Max HP / 100 = 486.233
Max HP = 48,623

Unless there was a DoT on it, there's no way its HP can break 48,623.

So while I'd normally be tempted to slap 50,000 on it because it's a nice, even number, it's just not possible. I'd be okay with 45,000 and assuming it got ~3000 HP restored from regen over the 12 minute fight.

Reply:
Makes sense and sounds reasonable indeed. Thank you again for laying it out that much. Will adjust.